introduction/theTechnology.md

```diff
@@ -0,0 +1,59 @@
+<a href="https://www.surfly.com/">![logo](../images/logosmall.png)</a>
```

Comment:
I noticed that there are no links to this page, so it is never rendered, not sure if it was the intention